tests/e2e/kcs/test_selinux_relabel_solution.py

```diff
@@ -0,0 +1,307 @@
+                f"get pod {pod_name} -n openshift-storage -o jsonpath='{{.status.conditions}}'"
+            )
+            conditions = [
+                {key: None if value == "null" else value for key, value in item.items()}
+                for item in conditions
+            ]
+
+            # Get lastTransitionTime for different condition type of pod
+            containers_ready_time = None
+            pod_scheduled_time = None
+            for condition in conditions:
+                if condition["type"] == "ContainersReady":
+                    containers_ready_time = datetime.strptime(
+                        condition["lastTransitionTime"], "%Y-%m-%dT%H:%M:%SZ"
+                    )
+                elif condition["type"] == "PodScheduled":
+                    pod_scheduled_time = datetime.strptime(
+                        condition["lastTransitionTime"], "%Y-%m-%dT%H:%M:%SZ"
+                    )
+
+            # Calculate the difference between ContainersReady and PodScheduled time
+            if containers_ready_time and pod_scheduled_time:
+                time_difference = containers_ready_time - pod_scheduled_time
+                return time_difference.total_seconds()
+
+        except Exception as e:
+            log.info(f"Error retrieving pod information for '{pod_name}': {e}")
+
+        return None
+
+    def teardown(self):
+        """
+        Cleanup the test environment
+        """
+        res_pod.delete_deploymentconfig_pods(self.pod_objs[0])
+        self.pvc_obj.delete()
+        time.sleep(30)
+        self.sa_name.delete()
+
+    @pytest.mark.parametrize("copies", [5])
+    def test_selinux_relabel(self, copies):
+        """
+        Steps:
+            1. Create cephfs pvcs and attach pod with more than 100K files across multiple nested directories
+            2. Take md5sum for them some random files and get pod restart time
+            3. Apply the fix/solution from kcs https://access.redhat.com/solutions/6906261
+            4. Restart the pods which are hosting cephfs files in large numbers.
+            5. Check data integrity.
+            6. Check for relabeling - this should not be happening.
+
+        Args:
+            copies: number of copies to write kernel files in pod
+
+        """
+        self.project_namespace = constants.OPENSHIFT_STORAGE_NAMESPACE
+
+        # Create cephfs pvc
+        self.pvc_obj = helpers.create_pvc(
+            namespace=self.project_namespace,
+            sc_name=constants.DEFAULT_STORAGECLASS_CEPHFS,
+            size="20Gi",
+        )
+
+        # Create deployment pod
+        self.pod_objs = self.create_deploymentconfig_pod(
+            command=["/opt/multiple_files.sh"],
+            command_args=[f"{copies}", "/mnt"],
+        )
+        log.info(f"files copied to pod {self.pod_objs.name}")
+        self.pod_selector = self.pod_objs.labels.get("deploymentconfig")
+        pod1_name = self.pod_selector + "-1-deploy"
```

Comment:
you already have a func created above to get the app pod name, please call it here